.github/workflows/ci_docs.yml

```diff
@@ -31,12 +31,11 @@ on:
 
 jobs:
   docs:
-    name: ${{ matrix.os }} - Python ${{ matrix.python-version }}
+    name: ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.11']
```

Comment:
I don't think we will build docs on multiple Python versions, so no need to define `python-version` in matrix.